tasks/create-internal-request/README.md

```diff
@@ -8,13 +8,21 @@ Creates an InternalRequest resource to call IIB service
 |------|-------------|----------|---------------|
 | pipelineRunName | The name of the Parent PipelineRun for this task | No | `ir-$(context.pipelineRun.name)` |
 | request | Internal pipeline request name | No | |
-| params | Internal Request parameters | No | |
+| internalRequestParams | Internal Request parameters | No | |
+| dataPath | Path to the JSON string of the merged data to use in the input workspace | Yes | "data.json" |
 | inputDataFile | Optional file to read data from | Yes | "" |
 | updateGenericResult | Should the task update the genericResult result  | Yes | "false" |
 | requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | 360 |
+| subdirectory | Subdirectory inside the workspace to be used | Yes | "" |
 
 ## Changelog
 
+### changes since 0.8
+- adds parameter `dataPath` to support data JSON file
+- adds step `merge-internal-request-parameters` to merge data from
+  data.json with the internal-request parameters to keep the compatibility
```

Comment:
"the compatibility" -> "backward compatibility"

But I wonder, why do we keep support for pipeline parameters if we're no longer gonna use it? Is there a reason for it?

Reply:
I will rewrite this phrase. It does not make sense.

Reply:
removed what concerns to compatibility

Reply:
OK, thanks. Any comment on why we still support pipeline params? Without it, we wouldn't have to merge anything. Also, while on this topic, why couldn't we simply use `jq` with the `*` operator to merge two jsons like we do in collect-data or apply-mapping? But again, if this task is going away soon anyway, there's probably no need to rewrite it now.

Reply:
Oh, so looking at https://github.com/redhat-appstudio/release-service-catalog/pull/240 , the parameters will still be used, but only for a few things, right?

Reply:
OK, thanks.